doc/modules/neural_networks_supervised.rst

```diff
@@ -270,7 +270,7 @@ function for classification is Cross-Entropy, which in binary case is given as,
 
 .. math::
 
-    Loss(\hat{y},y,W) = -y \ln {\hat{y}} - (1-y) \ln{(1-\hat{y})} + \alpha ||W||_2^2
```

Comment:
That's a bit unrelated no? But looks fine as well.

Reply:
This is unrelated. I think I commited this in the wrong branch. Reverting

Reply:
This was suppose to be for https://github.com/scikit-learn/scikit-learn/pull/22151